code/game/objects/items/mail.dm

```diff
@@ -107,26 +107,47 @@
 			var/tag = uppertext(GLOB.TAGGERLOCATIONS[destination_tag.currTag])
 			to_chat(user, span_notice("*[tag]*"))
 			sort_tag = destination_tag.currTag
-			playsound(loc, 'sound/machines/twobeep_high.ogg', 100, TRUE)
+			playsound(loc, 'sound/machines/twobeep_high.ogg', vol = 100, vary = TRUE)
+
+/obj/item/mail/multitool_act(mob/living/user, obj/item/tool)
+	. = FALSE // FALSE - means this proc was not stopped by return statement.
+	if(user.get_inactive_held_item() == src)
+		if(src.vars["armed"] == null || src.vars["armed"] == FALSE)
```

Comment:
What the
Why are you doing `vars`

Reply:
Because this variable may exist only on child type.
I wanted to evade copy-paste of code...

Reply:
I don't understand O_o

Reply:
Vars use is bad, you should check this on the subtype

Reply:
Okay